modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/ConversionUtils.scala

```diff
@@ -471,4 +473,8 @@ object ConversionUtils {
       nvps = r.asScala.toList
       pairs = nvps.map(p => p.getName() -> p.getValue())
     } yield pairs.toMap
+
+  /** Extract valid IP (v4 or v6) address from a string */
+  def extractInetAddress(arg: String): Option[InetAddress] =
```

Comment:
Would be great to add a unit test

Reply:
Yup, you're right, coveralls notified about dropped coverage and think this is the reason.